bot/__main__.py

```diff
@@ -7,37 +7,40 @@
 from bot.core.bot import Bot
 
 TOKEN = os.getenv("BOT_TOKEN")
+PREFIX = config.COMMAND_PREFIX
 
 extensions = [
     "bot.cogs.codesandbox",
+    "bot.cogs.coding",
     "bot.cogs.commands",
-    "bot.cogs.conversion",
+
+    "bot.cogs.comics",
     "bot.cogs.common",
+    "bot.cogs.conversion",
+    "bot.cogs.documentation",
+    "bot.cogs.embeds",
     "bot.cogs.emotes",
     "bot.cogs.events",
     "bot.cogs.fun",
     "bot.cogs.games",
+    "bot.cogs.github",
     "bot.cogs.moderation",
+    "bot.cogs.nasa",
+    "bot.cogs.reddit",
+    "bot.cogs.search",
+    "bot.cogs.security",
     "bot.cogs.study",
     "bot.cogs.sudo",
     "bot.cogs.support",
     "bot.cogs.tools",
-    "bot.cogs.search",
-    "bot.cogs.security",
-    "bot.cogs.embeds",
-    "bot.cogs.comics",
-    "bot.cogs.coding",
-    "bot.cogs.documentation",
-    "bot.cogs.reddit",
     "bot.cogs.translate",
-    "bot.cogs.github",
-    "bot.cogs.nasa",
 ]
 
+
 bot = Bot(
     extensions,
-    command_prefix=when_mentioned_or(config.COMMAND_PREFIX),
-    activity=Game(name=f"Ping me using {config.COMMAND_PREFIX}help"),
+    when_mentioned_or(PREFIX),
```

Comment:
I don't see a reason to remove the `command_prefix` kwarg and replace it with a regular arg, revert this change to use the `command_prefix= when_mentioned_or(PREFIX)`

Reply:
K